test/helpers/kubectl.go

```diff
@@ -1787,8 +1811,9 @@ func (kub *Kubectl) GetDefaultIface() (string, error) {
 func (kub *Kubectl) DeleteCiliumDS() error {
 	// Do not assert on success in AfterEach intentionally to avoid
 	// incomplete teardown.
-
-	_ = kub.DeleteResource("ds", fmt.Sprintf("-n %s cilium", GetCiliumNamespace(GetCurrentIntegration())))
+	ns := GetCiliumNamespace(GetCurrentIntegration())
```

Comment:
As you are changing this, please drop the `ns` var and use `CiliumNamespace` from `cons.go`. (Same package)